HospitalFrontend/src/views/DocSearch.vue

```diff
@@ -211,11 +264,38 @@ export default {
               backgroundColor: "linear-gradient(to right, #C37D92, #d89a9e)"
             }).showToast()
         },
+        handleExaminations(response) {
+            if (response.data && response.data.length) {
+              response.data.forEach(exam => {
+                console.log(exam);
```

Comment:
This is unnecessary. You left some comments as well.

Reply:
Removed.